Packet++/header/IPv4Layer.h

```diff
@@ -189,6 +187,9 @@ namespace pcpp
 		/** A list of IPv4 addresses parsed from the IPv4 timestamp option value */
 		std::vector<IPv4Address> ipAddresses;
 
+		/** The default constructor */
+		IPv4TimestampOptionValue() : type(IPv4TimestampOptionValue::Unknown) {}
```

Comment:
what is `type()`? is it compliant with old versions of c++?

Reply:
It is initialization of the structure's member not a keyword. There is the method `clear` which sets this member to `IPv4TimestampOptionValue::Unknown` but there is no a constructor which does the same.